e2e-tests/content-strings/pages/index.js

```diff
@@ -165,7 +165,7 @@ const COOKIES_SAVED_PAGE = {
   BODY: 'You can change your preferences at any time.',
 };
 
-const NEED_TO_START_AGAIN_PAGE = {
+const NEED_TO_START_AGAIN_EXIT_PAGE = {
```

Comment:
This has `EXIT_PAGE`, but others have `EXIT` - can we still to one or the other?

I think `_EXIT_PAGE` would be clearest because otherwise could seem a little strange e.g `NEED_TO_START_AGAIN_EXIT` could be interpreted differently